lib/archethic/contracts/interpreter/condition.ex

```diff
@@ -263,6 +263,14 @@ defmodule Archethic.Contracts.ConditionInterpreter do
     {node, acc}
   end
 
+  # Whitelist the get_first_address/0 function in condition
+  defp prewalk(
+         node = {{:atom, "get_first_transaction_address"}, _, []},
```

Comment:
There is a mismatch between comment and the real atom. You could use `get_first_address/0`